test/e2e/prometheus_test.go

```diff
@@ -57,6 +57,31 @@ var (
 	certsDir = "../../test/e2e/remote_write_certs/"
 )
 
+func createmTLSSecret(t *testing.T, secretName, ns string) {
```

Comment:
(nit) should it be `createMTLSSecret?

Reply:
Since its usually abbreviated `mTLS` I used same but can change to this :)